online distro logs warnings and an error in openhab.log during startup

Launching the online distro, I find the following in the openhab.log:

```
18:38:40.281 [INFO ] [.openhab.core.internal.CoreActivator] - openHAB runtime has been started (v2.0.0, build 201601060009).
18:39:27.736 [WARN ] [ce.jetty.internal.JettyServerWrapper] - failed to parse and instantiate of javax.servlet.ServletContainerInitializer in classpath
18:39:29.407 [WARN ] [ce.jetty.internal.JettyServerWrapper] - failed to parse and instantiate of javax.servlet.ServletContainerInitializer in classpath
18:39:37.836 [WARN ] [ce.jetty.internal.JettyServerWrapper] - failed to parse and instantiate of javax.servlet.ServletContainerInitializer in classpath
18:39:51.190 [WARN ] [ce.jetty.internal.JettyServerWrapper] - failed to parse and instantiate of javax.servlet.ServletContainerInitializer in classpath
18:41:38.282 [ERROR] [core.karaf.internal.FeatureInstaller] - Failed installing 'openhab-package-demo': Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=openhab-binding-yahooweather; type=karaf.feature; version=0 [caused by: Unable to resolve openhab-binding-yahooweather/0.8.0.SNAPSHOT: missing requirement [openhab-binding-yahooweather/0.8.0.SNAPSHOT] osgi.identity; osgi.identity=shk-esh-binding-yahooweather; type=karaf.feature]
```

--- a/distributions/openhab-online/src/main/resources/etc/org.ops4j.pax.url.mvn.cfg
+++ b/distributions/openhab-online/src/main/resources/etc/org.ops4j.pax.url.mvn.cfg
@@ -37,6 +37,23 @@ org.ops4j.pax.url.mvn.useFallbackRepositories=false
 #
 # org.ops4j.pax.url.mvn.proxySupport=false
 
+#
+# Comma separated list of repositories scanned when resolving an artifact.
+# Those repositories will be checked before iterating through the
+#    below list of repositories and even before the local repository
+# A repository url can be appended with zero or more of the following flags:
+#    @snapshots  : the repository contains snaphots
+#    @noreleases : the repository does not contain any released artifacts
+#
+# The following property value will add the system folder as a repo.
+#
+org.ops4j.pax.url.mvn.defaultRepositories=\
+    file:${karaf.home}/${karaf.default.repository}@id=system.repository@snapshots, \
+    file:${karaf.data}/kar@id=kar.repository@multi@snapshots, \
+    file:${karaf.base}/${karaf.default.repository}@id=child.system.repository@snapshots
+
+# Use the default local repo (e.g.~/.m2/repository) as a "remote" repo
+#org.ops4j.pax.url.mvn.defaultLocalRepoAsRemote=false
 
 #
 # Comma separated list of repositories scanned when resolving an artifact.

